MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -111,29 +112,33 @@ def fetch_github_issues(args, db):
 
 			#Get the last updated time for the repo
 			since_info = get_collection_record({'repository': repo}, None, 'LastUpdatedInfo', db)
-			
-			num_issues = 0
 			since = None
 
-			#Get the last_updated time and total_issues_collected if the repo is found
+			#Get the last_updated time if the repo is found
 			if since_info:
 				since = since_info['last_updated_time']
-				num_issues = since_info['total_issues_collected']
-	
+			
 			pprint(f"Fetching open issues for {repo}")
```

Comment:
The `open` here can be changed to `new`.

Reply:
Done! Sorry for missing out on this.